src/ONE.Models.CSharp/Enterprise/Twin/Wizard/Location.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONE.Models.CSharp.Enterprise.Twin.Wizard
+{
+    public class Location
```

Comment:
Isn't there already a location protobuf?

Reply:
Renamed to avoid confusion.